Services/Common/OJS.Services.Common/Implementations/SubmissionsDistributorCommunicationService.cs

```diff
@@ -118,7 +118,11 @@ private object BuildDistributorSubmissionBody(Submission submission)
                     CheckerType = checkerType,
                     CheckerParameter = submission.Problem.Checker?.Parameter,
                     Tests = tests,
-                    submission.Problem.SolutionSkeleton
+                    SolutionSkeleton = submission.Problem
```

Comment:
Update this to be `TaskSkeleton = ...` as this is the new name that the worker expects.